barretenberg/cpp/src/barretenberg/plonk/transcript/transcript.hpp

```diff
@@ -0,0 +1,122 @@
+#pragma once
+#include "manifest.hpp"
+#include <array>
```

Comment:
hmm wonder why it isn't showing as a move

Reply:
I think I did a bit of pruning of includes, stuff like this that confused the git mvs. I was strict in the others and they have very small diffs bc of better mv detection.